Scripts/Services/Conversation/DataModels.cs

```diff
@@ -282,7 +282,7 @@ public class SystemResponse
     /// <summary>
     /// An array of dialog node IDs that are in focus in the conversation.
     /// </summary>
-    public string[] dialog_stack { get; set; }
+    public DialogNode[] dialog_stack { get; set; }
```

Comment:
Conversation needs to be a <key,value> object. You can't create objects based on the existing JSON because the service is constantly changing it.

It's better if you have a Map/Dictionary and you use that to get/set properties